ftw/linkchecker/report_generating.py

```diff
@@ -0,0 +1,74 @@
+import xlsxwriter
+from cell_format import BOLD, CENTER
+import time
+
+
+class ReportCreator(object):
+
+    def __init__(self, xlsx_file_path):
+        self.xlsx_file_path = xlsx_file_path
+        self.row = 0
+        self.workbook = xlsxwriter.Workbook(self.xlsx_file_path)
+        self.worksheet = self.workbook.add_worksheet()
+        self.autofitter_list_collector = []
+
+    def append_report_data(self, report_data, format=None):
+        format = format.get_workbook_format(self.workbook) if format else None
+
+        for row_list in (report_data):
+            col = 0
+            for value in row_list:
```

Comment:
instead of `col` you can use python's `enumerate` inbuild` function

Reply:
That wouldn't work in that case. But since I changed quite a bit to make it better readable it doesn't appear anyway.

Reply:
Well ok... but enumerate usually does that job 😊